test/results/generate_02.ics

```diff
@@ -10,7 +10,7 @@ DTEND:20131004T231500Z
 SUMMARY:Sample Event
 LOCATION:localhost
 DESCRIPTION:Lorem ipsum dolor sit amet\, consetetur sadipscing elitr\, sed
-  diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam era
- t\, sed diam voluptua.\nbeep boop
+ diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam era
+  t\, sed diam voluptua.\nbeep boop

```

Comment:
Perfect. Now the CI tests should work well. I'm a bit busy today so I may not get around to it for a while, but I'll see if I can look at it a bit closer to see what needs to happen on the parsing side of things.